src/Items/ItemShears.h

```diff
@@ -68,7 +63,7 @@ class cItemShearsHandler :
 
 	virtual short GetDurabilityLossByAction(eDurabilityLostAction a_Action) override
 	{
-		return 0;
+		return 1;
```

Comment:
Yes (at least, for each block break).

Perhaps you're thinking of way back when, when shears could be enchanted with silk touch and efficiency and were usable as an infinite durability shovel.  That's no longer possible.